integration-cli/docker_cli_exec_test.go

```diff
@@ -339,6 +339,7 @@ func (s *DockerSuite) TestInspectExecID(c *check.C) {
 
 		out = strings.TrimSuffix(out, "\n")
 		if out != "[]" && out != "<no value>" {
+			c.Logf("out: %q", out)
```

Comment:
Does this log need to be here? Looks like you possibly had this here for test debugging?